### PR TITLE
Adding installation phantomjs to docker image

### PR DESCRIPTION
**Pull request by aleksbykov**
Phantomjs installed during docker image build process.
Code of phantomjs installation is left to save workaround install
phantomjs in sct.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@aleksbykov can you rebase this on master again ?

**Comment by aleksbykov** [comment]
@fruch done

File: sdcm/logcollector.py [hunk 1]
```diff
@@ -340,10 +340,10 @@ def install_phantom_js(self):
                 tar xvfj {phantomjs_tar}
             """.format(**locals()))
             localrunner.run("bash -ce '%s'" % install_phantom_js_script)
-            localrunner.run(
-                "cd {0.phantomjs_base} && sed -e 's/200);/10000);/' examples/rasterize.js |grep -v 'use strict' > r.js".format(self))
         else:
             LOGGER.debug("PhantomJS is already installed!")
+        localrunner.run(
+            "cd {0.phantomjs_base} && sed -e 's/200);/10000);/' examples/rasterize.js |grep -v 'use strict' > r.js".format(self))
```
Comment (amoskong) [inline sdcm/logcollector.py]: I removed the 'use strict' for workaround an error in generating the snapshot.
The issue had been fixed by https://github.com/ariya/phantomjs/pull/14824.

However the sed for increasing timeout is still needed, it's very slow to load all metrics.

Comment (amoskong) [reply sdcm/logcollector.py]: But there is no official release of phantomjs after 2016-01-24, maybe we can save fixed examples/rasterize.js to data_dir/ like `data_dir/share_snapshot.js`?

Comment (fruch) [reply sdcm/logcollector.py]: Sounds reasonable, until we can replace phantomjs if it's not clearly maintained

**Comment by bentsi** [comment]
@aleksbykov This PR didn't solve the main issue. See details in the Trello task
https://trello.com/c/mo81v2By/1318-dont-install-phanotom-js-on-each-hydra-command
@fruch we should understand the context of the PR before merging it

**Pull request merged**